audit/get-active-sandbox-and-space-count.sh

```diff
@@ -8,7 +10,38 @@
 # We can determine this by using the cf CLI and CAPI to retrieve all sandbox
 # orgs, then filter a list of all apps with those corresponding org GUIDs.
 
+if [[ $1 == "--help" ]]; then
+  echo " "
+  echo "Counts the number of current active sandboxes in the platform."
+  echo " "
+	echo "Usage: get-active-sandbox-and-space-count [START_DATE] [END_DATE]"
+  echo " "
+  echo -e "  START_DATE: \t A date to limit finding active sandboxes at or after the provided date in YYYY-MM-DD format."
```

Comment:
the language is inconsistent, so I'm not sure the intended behavior here. Are we looking for:
- sandboxes created between the dates
- sandboxes that were active at any time between the dates

Reply:
Ah good catch - the script is doing a slightly different thing if a date is supplied, so let me clarify and see if that helps.  It may also be easier/better to just split the behavior apart into separate scripts.

All told, on the one hand have the ability to count all current active sandboxes.  Supplying dates is somewhat immaterial because sandboxes can only be active for 90 days at most and are then cleared out.

When you do supply dates though, this checks for how many spaces were created in sandbox organizations during that time period (or at least limited to on or after a date, or before and on a date if only one is supplied).

Reply:
@bengerman13 this should be a lot better now.  I think we can still have everything in here but the language, instructions, and intent should be much clearer!

Reply:
the updated comment helps, but I think the help message is still unclear

Reply:
I'll tweak it further, thanks!

Reply:
Finally got the changes committed, this should be a lot better now!